CONTRIBUTING.md

```diff
@@ -1,8 +1,19 @@
-# How to Contribute
+# How to contribute
+First, thanks for taking the time to contribute to our project! There are many ways you can help out.
 
-*Read this in other languages: [中文](translations/zh/CONTRIBUTING.md).*
+### Questions
 
-When submitting a pull request (PR), please use the following guidelines:
+If you have a question that needs an answer, [create an issue](https://help.github.com/articles/creating-an-issue/), and label it as a question.
+
+### Issues for bugs or feature requests
+
+If you encounter any bugs in the code, or want to request a new feature or enhancement, please [create an issue](https://help.github.com/articles/creating-an-issue/) to report it. Kindly add a label to indicate what type of issue it is.
```

Comment:
Should we use this link `https://github.com/yahoo/elide/issues/new` instead of `https://help.github.com/articles/creating-an-issue/`